qemu/tests/blockdev_snapshot_chains.py

```diff
@@ -0,0 +1,109 @@
+import logging
+
+from virttest import data_dir
+from virttest import error_context
+
+from provider.blockdev_snapshot_base import BlockDevSnapshotTest
+from provider.virt_storage.storage_admin import sp_admin
+
+
+class BlockdevSnapshotChainsTest(BlockDevSnapshotTest):
+
+    def __init__(self, test, params, env):
+        self.snapshot_num = int(params.get("snapshot_num"))
```

Comment:
default value is required here,  convert None to int will raise exception.

Reply:
Will update it later, thanks.